lib/display.rb

```diff
@@ -0,0 +1,14 @@
+class Display
+  def display_grid(grid)
+    print """
+     #{grid[0]} | #{grid[1]} | #{grid[2]}
+    -----------
+     #{grid[3]} | #{grid[4]} | #{grid[5]}
+    -----------
+     #{grid[6]} | #{grid[7]} | #{grid[8]}\n"""
+  end
+
+  def display_make_move
```

Comment:
I think that `display_make_move` is a shortcut name, as `Display` doesn't make moves. Why not go with `ask_for_move` or something of this kind?

Reply:
Thanks, Justyna. ask_for_move is a better name for this method